launcher/cli/main.go

```diff
@@ -41,6 +41,7 @@ func Main() {
 	RootCmd.PersistentFlags().StringP("data-dir", "d", "./dfuse-data", "Path to data storage for all components of dfuse")
 	RootCmd.PersistentFlags().StringP("config-file", "c", "./dfuse.yaml", "dfuse configuration file to use")
 	RootCmd.PersistentFlags().String("nodeos-path", "nodeos", "Path to the nodeos binary. Defaults to the nodeos found in your PATH")
+	initCmd.PersistentFlags().Bool("skip-checks", false, "Skip checks to ensure 'nodeos' binary is supported")
```

Comment:
`--skip-nodeos-version-checks` ? c'est une top-level global.. `skip-checks` pourrait s'appliquer confusément ailleurs j'pense.

Reply:
I agreed, I envision other checks in the future maybe like system restrictions and maybe others, that's why I named it kind of "global".

But I agree, I'll check to rename the flag is it was released like this. `skip-nodeos-checks` would probably better than version.